thirdparty/bullet/BulletDynamics/Featherstone/btMultiBody.cpp

```diff
@@ -3,7 +3,7 @@
  *   Class representing an articulated rigid body. Stores the body's
  *   current state, allows forces and torques to be set, handles
  *   timestepping and implements Featherstone's algorithm.
- *   
+ *
```

Comment:
Your IDE removed trailing spaces in thirdparty code, could you remove those changes?

Reply:
Yes